init/init

```diff
@@ -56,6 +56,8 @@ do
         esac
 done < /proc/cgroups
 
+/sbin/ip link set lo up
```

Comment:
Does anyone have issues doing it this way? It seems that for POD's we need the loopback interface to be up which is required to be done in the host namespace and rather than coding that in GCS we can just do it here.